portfolio/src/main/webapp/script.js

```diff
@@ -158,11 +158,13 @@ async function checkLogin(){
 
 /* Creates a map and adds it to the page. */
 function initMap() {
+  const JAKARTA = {lat: -6.175540, lng: 106.82743};
+  const ZOOM_SCALE = 5;
```

Comment:
nit: please use `lowerCamelCase` for constants in a function scope. [ref](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names)